App1/App1.csproj

```diff
@@ -68,6 +78,7 @@
 	<ItemGroup>
 		<PackageReference Include="Esri.ArcGISRuntime" Version="200.5.0" />
 		<PackageReference Include="Esri.ArcGISRuntime.Maui" Version="200.5.0" />
-		<PackageReference Include="VertiGIS.Mobile" Version="5.34.0.284" />
+		<!--PLACEHOLDER: Swap with 5.35 release before PR merges -->
+		<PackageReference Include="VertiGIS.Mobile" Version="5.34.0.284" /> 
```

Comment:
Is it necessary to do this for this PR? Usually we do update the Mobile SDK version along with every release, so I'm wondering if that can still be a separate story/PR.

Reply:
I know there was that null-check you were going to add in the SecurityService to fix the sign-in cancellation issue, but I think that can just not be part of this story and be fixed in the next Mobile-update PR.

Reply:
Oh, I see. Ok let's just do it all in this one PR then.